.github/workflows/ci.yaml

```diff
@@ -63,19 +63,22 @@ jobs:
 
   run-notebook-docker-image-builder:
     uses: ./.github/workflows/notebook_docker_image_builder.yaml
-    needs: approval-for-notebook-tests
+    # needs: approval-for-notebook-tests
```

Comment:
The changes in this file have been made to get early feedback from CI without additional manual interaction. 

I will revert most changes to this file as soon as review approval comes into reach.